From 833c56149edfaee8c1d922a4d76c618dac51426d Mon Sep 17 00:00:00 2001
From: "David.Wu" <david.wu@rock-chips.com>
Date: Fri, 7 Dec 2018 14:51:29 +0800
Subject: [PATCH] usb: dwc3: Change the dwc3 driver to dwc3-rockchip for rk1808
 and rk3399pro-npu

Use the dwc3-rockchip driver to auto switch the host/peripheral dr_mode.

Change-Id: I497570fad4caaffc9da08086f2758653afb729db
Signed-off-by: David.Wu <david.wu@rock-chips.com>
---
 drivers/usb/dwc3/dwc3-of-simple.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/usb/dwc3/dwc3-of-simple.c b/drivers/usb/dwc3/dwc3-of-simple.c
index c045b282e204..974335377d9f 100644
--- a/drivers/usb/dwc3/dwc3-of-simple.c
+++ b/drivers/usb/dwc3/dwc3-of-simple.c
@@ -162,7 +162,6 @@ static const struct dev_pm_ops dwc3_of_simple_dev_pm_ops = {
 static const struct of_device_id of_dwc3_simple_match[] = {
 	{ .compatible = "qcom,dwc3" },
 	{ .compatible = "xlnx,zynqmp-dwc3" },
-	{ .compatible = "rockchip,rk1808-dwc3" },
 	{ /* Sentinel */ }
 };
 MODULE_DEVICE_TABLE(of, of_dwc3_simple_match);
-- 
2.35.3

